tests/unittests/CMakeLists.txt

```diff
@@ -156,7 +156,7 @@ add_executable(GradCheckTest
 target_link_libraries(GradCheckTest
                       PRIVATE
                         Base
-                        ExecutionEngine
+                        ExecutionEngine2
```

Comment:
I should have noticed this sooner, but I don't think EE2 should be a separate library.  It doesn't follow our 1:1 library to lib/ subdirectory model.

Reply:
Are you ok with it being there only temporarily? Once everything is ported to EE2 there will be a mass rename to EE and EE2 will replace EE bringing the library count back to one.